tests/dashbio_demos/dash-alignment-viewer/layout_helper.py

```diff
@@ -0,0 +1,109 @@
+import base64
```

Comment:
This file should also be added to the app directory *at* the deployment step instead of having copies of it in every single app's folder. Maybe a CircleCI command to copy this file into the folder that is being deployed?

Reply:
The problem here is that you cannot do relative imports above the path you are in, so moving this up  will _definitely_ break the `index.py`. If we are fine with getting rid of that fake gallery index, then this is trivial, but if not, this won't do.

Also, this will make it harder for folks to copy out the sample app into their own apps since they'll need to remember to copy this file from a different place. Probably not a huge deal since this was already necessary.